src/masternode-budget.h

```diff
@@ -515,6 +520,17 @@ class CBudgetProposal
 
     void CleanAndRemove(bool fSignatureCheck);
 
+    // If URL doesn't have a scheme, add one
+    std::string GetFixedupURL(std::string scheme="http")
+    {
+        std::string ret=strURL;
+        if (!regex_match(strURL, URLRegEx))
+        {
+           ret = scheme+"://"+strURL;
```

Comment:
nit: missing 1 space at the beginning of the line.

Reply:
corrected; rebased and squashed (I think :)